PatternPal/PatternPal.Extension/Commands/SubscribeEvents.cs

```diff
@@ -1,21 +1,28 @@
 ﻿#region
 
 using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
+using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.VisualStudio.Shell;
-using PatternPal.Extension.Grpc;
 using EnvDTE;
 using EnvDTE80;
+using Microsoft.VisualStudio.Shell;
+using PatternPal.Extension.Grpc;
 using PatternPal.Protos;
-using System.Threading;
-using System.Collections.Generic;
-using System.Linq;
 
 #endregion
 
 namespace PatternPal.Extension.Commands
 {
+    public enum ServerStatusCodes
```

Comment:
Shouldn't an `enum` be automatically generated from the `proto`-definition? If you are using a differently defined `enum` here, it might not be the best idea to have them equally named.